### PR TITLE
Results

### PR DESCRIPTION
**Pull request by kairstenfay**
Add a /results page to the Y3 Website.

Depends on https://github.com/nkrumm/securelink/pull/41.

### CODE REVIEW

**Comment by kairstenfay** [comment]
[Connect seattleflu.org/results to securelink](https://trello.com/c/9EOAJeIj/791-connect-seattlefluorg-results-to-securelink)

File: public/results/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          <li>
+            <a href="/results">Results</a>
+          </li>
+        </menu>
+      </nav>
+    <h1>Seattle Flu Study</h1>
+    <img alt="Image with text that says 'powered by the' plus the Seattle Flu Study logo"
+      src="../images/logo_SFS.svg">
+  </header>
+
```
Comment (joverlee521) [inline public/results/index.html:34]: I think we want to include the text that is currently on https://scanpublichealth.org/results , just replace "SCAN" with "Seattle Flu Study"?

File: public/error.html [hunk 2]
```diff
@@ -11,6 +11,22 @@
...
+          <a href="/">Home</a>
+        </li>
+        <li>
+          <a href="/uw">Husky Coronavirus Testing</a>
+        </li>
+        <li>
+          <a href="https://scanpublichealth.org">SCAN</a>
+        </li>
+        <li>
+          <a href="/results">Results</a>
```
Comment (tsibley) [inline public/error.html]:
```suggestion
          <a href="/results">Look up your result</a>
```

File: public/index.html [hunk 3]
```diff
@@ -11,6 +11,22 @@
...
+          <a href="/">Home</a>
+        </li>
+        <li>
+          <a href="/uw">Husky Coronavirus Testing</a>
+        </li>
+        <li>
+          <a href="https://scanpublichealth.org">SCAN</a>
+        </li>
+        <li>
+          <a href="/results">Results</a>
```
Comment (tsibley) [inline public/index.html]:
```suggestion
          <a href="/results">Look up your result</a>
```

File: public/results/index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            <a href="/">Home</a>
+          </li>
+          <li>
+            <a href="/uw">Husky Coronavirus Testing</a>
+          </li>
+          <li>
+            <a href="https://scanpublichealth.org">SCAN</a>
+          </li>
+          <li>
+            <a href="/results">Results</a>
```
Comment (tsibley) [inline public/results/index.html]:
```suggestion
            <a href="/results">Look up your result</a>
```

File: public/results/index.html [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    <h1>Seattle Flu Study</h1>
+    <img alt="Image with text that says 'powered by the' plus the Seattle Flu Study logo"
+      src="../images/logo_SFS.svg">
+  </header>
+
+    <h2>Access Your Results</h2>
+  <button
+    onclick="window.location.href='https://securelink.labmed.uw.edu/seattleflu'"">
+    Click here to access your test results
+  </button>
```
Comment (tsibley) [inline public/results/index.html]: My (vague) understanding is that using a <button> + onclick is unfriendly to accessibility software. The more conventional approach is to stick to an <a> but then style it like a button. (Frameworks like Bootstrap come pre-built with such styles.)

File: public/uw/index.html [hunk 6]
```diff
@@ -11,6 +11,22 @@
...
+          <a href="/">Home</a>
+        </li>
+        <li>
+          <a href="/uw">Husky Coronavirus Testing</a>
+        </li>
+        <li>
+          <a href="https://scanpublichealth.org">SCAN</a>
+        </li>
+        <li>
+          <a href="/results">Results</a>
```
Comment (tsibley) [inline public/uw/index.html]:
```suggestion
          <a href="/results">Look up your result</a>
```

**Review by tsibley — CHANGES_REQUESTED** [review]

File: public/results/index.html (hunk 5 above)
Comment (kairstenfay) [reply public/results/index.html]: Gotcha, that's good to know. I'll add a commit that improves accessibility.

Comment (kairstenfay) [reply public/results/index.html]: Addressed in a new commit.

File: public/results/index.html (hunk 1 above)
Comment (kairstenfay) [reply public/results/index.html:34]: I actually just added this landing page as an afterthought after deciding not to make /results redirect to SecureLink. I'm awaiting a language revision from Misja for the new SFS resuts page.

Comment (kairstenfay) [reply public/results/index.html:34]: The language has been revised in a new commit.

Comment (joverlee521) [reply public/results/index.html:34]: Thanks for updating!

File: public/style.css [hunk 7]
```diff
@@ -35,12 +34,15 @@ button {
     font-size: 22px;
     font-weight: 700;
     cursor: pointer;
+    margin: 0 auto;
```
Comment (joverlee521) [inline public/style.css:37]: I would add a `max-width` or `width: fit-content` so the button doesn't span the entire page.

Comment (kairstenfay) [reply public/style.css:37]: Good idea. I didn't know about `fit-content()`. According to MDN, it's a [combination of min() and max()](https://developer.mozilla.org/en-US/docs/Web/CSS/fit-content). Unfortunately as I learned recently, `min()` isn't supported on all browsers, so I'll stick with `max-width` for now.

Comment (kairstenfay) [reply public/style.css:37]: Added in a new commit.

**Pull request merged**